pages/art-pieces/index.js

```diff
@@ -0,0 +1,11 @@
+import ArtPiecePreview from "@/components/ArtPiecePreview";
+
+export default function ArtPieces({ artPieces }) {
+  return (
+    <>
+      {artPieces.map((artPiece) => {
+        return <ArtPiecePreview key={artPiece.slug} artPiece={artPiece} />;
```

Comment:
hier haben wir eine ul gemacht und dann jedes einzelne artPiece in ein li-Element gepackt. Das hatten wir in den Sessions so gemacht.